### ISSUE TITLE
radio-button: Focus Indicator and Selection not Visible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by kittenish** [angular/material#11975]
## Bug

# Demo and steps to reproduce the issue
[This blank StackBlitz demo](https://stackblitz.com/edit/angularjs-material-blank?file=app%2Fapp.template.html) can be used to create a reproduction that demonstrates your issue.

**Demo URL (required)***:  https://material.angularjs.org/HEAD/demo/radioButton

#### Detailed Reproduction Steps
1. Enable [Windows High Contrast](https://support.microsoft.com/en-us/help/13862/windows-10-use-high-contrast-mode) - Black theme
2. Navigate to a radio group using keyboard tab key
3. Observe the focus indicator is not visible
4. Select a radio button with arrow keys or mouse
5. Observe the selected state is not conveyed visually for the user

### Explain the expected behavior
In step 3, focus indicator should be visible.  
In step 5, selection should be visible.

### Explain the current behavior
Focus indicator and selection are not visible.

### Discuss the use-case or motivation for changing the existing behavior
It will be difficult to use under Windows high contrast if neither focus nor selection is visible.

### List the affected versions of AngularJS, Material, OS, and browsers

- AngularJS: 1.
- AngularJS Material: 1.
- OS:  Windows 10 Enterprise (64-bit)
- Browsers: Firefox 79.0.b6

#### Screenshots
A screenshot on Windows with high contrast mode:
![radioButtonHighContrast](https://user-images.githubusercontent.com/18439611/88327801-0f0c4800-cced-11ea-8bdf-0b97b1f66e1c.PNG)
